supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkJobExecutionActor.scala

```diff
@@ -172,7 +174,7 @@ class SparkJobExecutionActor(override val jobDescriptor: BackendJobDescriptor,
         SparkCommands.ExecutorCores -> runtimeAttributes.executorCores,
         SparkCommands.ExecutorMemory -> runtimeAttributes.executorMemory.toString.replaceAll("\\s","").toLowerCase,
         SparkCommands.AdditionalArgs -> runtimeAttributes.additionalArgs.getOrElse(""),
-        SparkCommands.SparkAppWithArgs -> command.get,
+        SparkCommands.SparkAppWithArgs -> command.toTry.get,
```

Comment:
I'm gonna flag this one too since it isn't in a `Spec`

Reply:
This is existing tech debt.